index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <meta charset=“UTF-8“ />
+    <title></title>
+    <link rel="stylesheet" href="styles/styles.css" />
+    <link href="https://fonts.googleapis.com/css?family=Anton|Questrial|Roboto" rel="stylesheet">
+
+
+
+  </head>
+  <body class="firmware-body">
+
```

Comment:
Please remove this empty lines too. Usually, we use only 1 empty line to separate components.